specs/BrowserProcessExited.md

```diff
@@ -66,25 +78,46 @@ void AppWindow::CloseWebView(/* ... */) {
               ICoreWebView2Environment* sender,
               ICoreWebView2BrowserProcessExitedEventArgs* args) {
           COREWEBVIEW2_BROWSER_PROCESS_EXIT_KIND kind;
+          UINT32 pid;
           CHECK_FAILURE(args->get_BrowserProcessExitKind(&kind));
-
-          // Watch for graceful browser process exit. Let ProcessFailed event
-          // handler take care of failed browser process termination.
-          if (kind == COREWEBVIEW2_BROWSER_PROCESS_EXIT_KIND_NORMAL)
+          CHECK_FAILURE(args->get_BrowserProcessId(&pid));
```

Comment:
Does the system keep the process handle open for the lifetime of this event (to prevent PID recycling)? Otherwise, we cannot guarantee that "the PID of the new browser process will be different to the PID of the older process".

Reply:
Sorry, I marked an earlier discussion around this as resolved, but it's still relevant here:

> The new browser process can't start with the same PID because the environment instance tracking the old process keeps a handle open and so the process object stays valid; its PID can't be taken until the environment instance observing it closes the handle, which happens only after the queued event gets processed. [...]